.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+
+    services:
+      toxiproxy:
+        image: shopify/toxiproxy
+        ports:
+          - 8474:8474
```

Comment:
I wish this could be solved better. In the Travis CI case we let Docker use the "host network" (i.e. it can open ports on the host) but as I understand it you can't do that on GitHub Actions, at least not in this net way of using Docker

Reply:
Travis reference: https://github.com/twingly/twingly-http/blob/7603cf9ad3e4c89cc147578f54f7f031b70f0488/.travis.yml#L8

and just to be clear, I don't like having to put the port there :) However, we do it in many other Docker scenarios, but then it's like well-known services as Memcached/MySQL etc where one can easily find the default ports with 3 seconds of Googling. Here one has to find the Toxiproxy file in the repository to figure out the port

Reply:
Maby add a comment explaining where the port comes from, but other than that I have no ideas.

Reply:
Yeah, I will look into it a bit more and see if it can be solved better

Reply:
This is the reply I found yesterday https://github.community/t/mysql-connection-from-docker-action/17094/5

Reply:
Made it a bit better in bbce4c0 I will leave it for now